apps/ledger-live-mobile/src/newArch/features/Accounts/Navigator.tsx

```diff
@@ -17,25 +17,51 @@ import SelectAccounts from "./screens/SelectAccounts";
 import AddAccountsWarning from "./screens/AddAccountWarning";
 import NoAssociatedAccountsView from "./screens/NoAssociatedAccountsView";
 import CloseWithConfirmation from "LLM/components/CloseWithConfirmation";
-import { StackNavigatorNavigation } from "~/components/RootNavigator/types/helpers";
-import { BaseNavigatorStackParamList } from "~/components/RootNavigator/types/BaseNavigator";
+import {
+  BaseComposite,
+  StackNavigatorNavigation,
+  StackNavigatorProps,
+} from "~/components/RootNavigator/types/helpers";
 import { NavigationHeaderCloseButtonAdvanced } from "~/components/NavigationHeaderCloseButton";
 
+import useAnalytics from "LLM/hooks/useAnalytics";
+import { AddAccountsNavigatorParamList } from "~/components/RootNavigator/types/AddAccountsNavigator";
+type NavigationProps = BaseComposite<
+  StackNavigatorProps<AddAccountsNavigatorParamList, NavigatorName.AddAccounts>
+>;
 export default function Navigator() {
   const { colors } = useTheme();
-  const route = useRoute();
+  const route = useRoute<NavigationProps["route"]>();
   const accountListUIFF = useFeature("llmAccountListUI");
-  const navigation = useNavigation<StackNavigatorNavigation<BaseNavigatorStackParamList>>();
+  const navigation = useNavigation<StackNavigatorNavigation<AddAccountsNavigatorParamList>>();
+
+  const { analyticsMetadata } = useAnalytics("addAccounts");
```

Comment:
Use the `AnalyticContexts.AddAccounts` everywhere instead of the `"addAccounts"` string